### PR TITLE
Remove alt-arch config items that are not needed any more

### PR DESCRIPTION
**Pull request by hhorak**
Related to https://github.com/sclorg/mongodb-container/pull/320 and https://github.com/sclorg/postgresql-container/pull/388 that remove the non-intel specific imagestreams.

### CODE REVIEW

**Comment by hhorak** [comment]
@yselkowitz Running `make verify` did not work for me, so I'd welcome feedback if this is enough to be done once sclorg/mongodb-container#320 and sclorg/postgresql-container#388 are merged or something more needs to be done.

**Comment by gabemontero** [comment]
> @yselkowitz Running `make verify` did not work for me, so I'd welcome feedback if this is enough to be done once [sclorg/mongodb-container#320](https://github.com/sclorg/mongodb-container/pull/320) and [sclorg/postgresql-container#388](https://github.com/sclorg/postgresql-container/pull/388) are merged or something more needs to be done.

it will a few hours from now @hhorak but I can check this PR out and try `make verify` as well ... based on what was deleted, I would think it would have worked.

@coreydaley FYI

**Comment by coreydaley** [comment]
So, it looks like you need to run make import locally and commit the changes.  If you add or remove stuff from official.yaml, that is going to change what imagestreams and templates are present.

**Review by yselkowitz — CHANGES_REQUESTED** [review]
Arch tags are missing, and make import needs to be run and its results added to the commit.  Then make verify should pass.

**Comment by gabemontero** [comment]
thanks @coreydaley @yselkowitz for beating me to the punch and sorting our the additional step @hhorak needs to execute

**Comment by hhorak** [comment]
I should have shared more info right from beginning -- `make import` does not work because I don't have `./library` binary and when trying to build it, `make build` fails with:
```
$ make build
go version go1.14.4 gccgo (GCC) 10.2.1 20200723 (Red Hat 10.2.1-1) linux/amd64
go build -mod=vendor -race
package github.com/openshift/library
	imports runtime/race: cannot find package
make: *** [Makefile:34: build] Error 1
```
I did run `make vendor` before, which succeeded, but it doesn't help. I'm not a golang developer, so I might miss some basic thing here. Any hint?

**Comment by yselkowitz** [comment]
Looks like you have gccgo instead of golang as your /usr/bin/go.  Either use golang, or just run `go build -mod=vendor` (without `-race`) manually.

**Comment by yselkowitz** [comment]
@hhorak if you `git pull --rebase` now, make build might work with gccgo now.

**Comment by hhorak** [comment]
I installed golang and it worked as well, thanks for the suggestion. The PR is now updated with the imported files. There is one more PR not merged (https://github.com/sclorg/django-ex/pull/164) but I guess it doesn't block this PR, because it will be updated by the script once merged. Hope this PR is good now.

**Comment by hhorak** [comment]
Hups, sorry, arch tags are still missing, fixing right away..

File: official.yaml [hunk 1]
```diff
@@ -361,20 +361,6 @@ data:
           - ocp
           - online-starter
           - online-professional
```
[the remark this answers is not in the document]
Comment (hhorak) [reply official.yaml:363]: Fixed now.

File: official.yaml [hunk 2]
```diff
@@ -460,20 +446,6 @@ data:
...
```
[the remark this answers is not in the document]
Comment (hhorak) [reply official.yaml:451]: Fixed now.

**Comment by hhorak** [comment]
> Hups, sorry, arch tags are still missing, fixing right away..

Fixed now.

**Comment by gabemontero** [comment]
/hold cancel

/approve 

So assuming the test jobs pass now, I see deletes.  And when I look at some of the files htat were changed, I still see the *-ubi8 and *-ubi7 pattern that has been recently established.

That said, while I've approved this, if possible @yselkowitz since you are most familiar with our shift here, I'd like for you to give this the final review.  If you are good, I believe with my approve, you lgtm (using / to add the label) should be sufficient to merge this.

thanks everyone

**Comment by gabemontero** [comment]
and the tests passed while I was typing the last comment :-)

**Comment by yselkowitz** [comment]
Thanks Honza this looks good now.
/lgtm

**Pull request merged**